src/annotator/test/adder-test.js

```diff
@@ -24,6 +31,35 @@ function revertOffsetElement(el) {
   el.style.top = '0';
 }
 
+describe('findHighestZindex', () => {
+  it('returns zero z-index (default value) if no element is passed)', () => {
+    assert.strictEqual(findHighestZindex(undefined), 0);
+    assert.strictEqual(findHighestZindex(null), 0);
+  });
+
+  it('returns z-index', () => {
+    const wrapper = mount(
+      <div className="parent" style={{ zIndex: 10 }}>
+        <div className="dummyChild" style={{ zIndex: 20 }} />
+        <div className="child1" style={{ zIndex: 5 }} />
+        <div className="child2" style={{ zIndex: 15 }} />
+      </div>,
+      { attachTo: document.body }
+    );
+
+    const parent = wrapper.find('.parent').getDOMNode();
+    assert.strictEqual(findHighestZindex(parent), 10);
+
+    const child1 = wrapper.find('.child1').getDOMNode();
+    assert.strictEqual(findHighestZindex(child1), 10);
+
+    const child2 = wrapper.find('.child2').getDOMNode();
+    assert.strictEqual(findHighestZindex(child2), 15);
+
+    wrapper.detach();
```

Comment:
If you use the suggestion I mentioned in the thread about creating a container and then removing it in an `afterEach` callback, then the `.detach()` call can be removed here. Calling `wrapper.unmount()` is mainly needed when there are effects that need to be cleaned up, but that is not the case with this simple JSX.

Reply:
I found that rendering components directly into `document.body` is discouraged using enzyme

Reply:
Both `wrapper.detach()` and `wrapper.unmount()` can be used if the component is not attached directly to `document.body`.